api-inference-community/docker_images/pyannote/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM tiangolo/uvicorn-gunicorn:python3.8
+LABEL maintainer="me <me@example.com>"
+
+# Add any system dependency here
+# RUN apt-get update -y && apt-get install libXXX -y
```

Comment:
I think you need to add `ffmpeg` here.

To check you can test the docker test locally.

```bash
RUN_DOCKER_TESTS=1 pytest -sv tests/test_dockers::DockerImageTests::test_pyannote
```